docs/source/usersguide/random_ray.rst

```diff
@@ -0,0 +1,303 @@
+.. _random_ray:
+
+=================
+Random Ray Solver
+=================
+
+In general, the random ray solver mode uses most of the same settings and :ref:`run strategies <usersguide_particles>` as the standard Monte Carlo solver mode. For instance, random ray solves are also split up into :ref:`inactive and active batches <usersguide_batches>`. However, there are a couple of settings that are unique to the random ray solver, and a few areas that the random ray run strategy differs, which will be described in this section.
+
+------------------------
+Enabling Random Ray Mode
+------------------------
+
+To utilize the random ray solver, the ``settings.random_ray`` dictionary must be present in the :class:`openmc.Settings` Python class. There are a number of additional settings that must be specified within this dictionary that will be discussed below. Additionally, the multi-group energy mode must be specified. 
+
+-------
+Batches
+-------
+
+In Monte Carlo, inactive batches are used to let the fission source develop into a stationary distribution before active batches are performed that actually accumulate statistics. While this is true of random ray as well, in the random ray mode the inactive batches are also used to let the scattering source develop. Monte Carlo fully represents the scattering source within each iteration (by its nature of fully simulating particles from birth to death through any number of physical scattering events), whereas the scattering source in random ray can only represent as many scattering events as batches have been completed. E.g., by iteration 10 in random ray, the scattering source only captures the behavior of neutrons through their 10th scatter. By iteration 500 in random ray, the scattering source only captures the behavior of neutrons through their 100th scatter. Thus, while inactive batches are only required in an eigenvalue solve in Monte Carlo, inactive batches are required for both eigenvalue and fixed source solves in random ray due to this additional need to converge the scattering source.
+
+The additional burden of converging the scattering source generally results in a higher requirement for the number of inactive batches - often by an order of magnitude or more. For instance, it may be reasonable to only use 50 inactive batches for a light water reactor simulation with Monte Carlo, but  random ray might require 500 or more inactive batches. Similar to Monte Carlo, :ref:`Shannon entropy
```

Comment:
It's probably worth pointing out here that this implies inactive batches have to be used even in fixed source problems!

Reply:
The paragraph currently makes this point already: "Thus, while inactive batches are only required in an eigenvalue solve in Monte Carlo, inactive batches are required for both eigenvalue and fixed source solves in random ray due to this additional need to converge the scattering source."

Also -- fixed source mode is not supported in random ray for this PR, but a fixed source PR is already done and will be submitted right after this one is merged.